test/manipulation.test.js

```diff
@@ -967,6 +967,13 @@ describe('manipulation', function() {
         });
       });
 
+      function changePostIdInBeforeSaveHook(operationHook, ctxField) {
```

Comment:
Please put helper functions at the bottom of the test file, see https://github.com/strongloop-internal/loopback-knowledge-base/blob/master/style-guide.md#layout-of-test-files